lib/ansible/plugins/callback/logstash.py

```diff
@@ -37,26 +43,29 @@
 import json
 import socket
 import uuid
-from datetime import datetime
-
 import logging
+from datetime import datetime
 
 try:
     import logstash
     HAS_LOGSTASH = True
 except ImportError:
     HAS_LOGSTASH = False
 
+try:
+    from __main__ import cli
+except ImportError:
+    # using API w/o cli
+    cli = False
```

Comment:
you should not need to do this, it already happens in teh base class

Reply:
done